script/generate-types

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
Not for now, but we should probably have [Shellcheck](https://www.shellcheck.net/) running as part of our CI pipeline, as we're using a lot of shell scripts now

Reply:
Yep good idea, I've made a note of it